src/components/TextInput/BaseTextInput.js

```diff
@@ -242,17 +242,16 @@ function BaseTextInput(props) {
     For other platforms, explicitly remove `lineHeight` from single-line inputs
     to prevent long text from disappearing once it exceeds the input space.
     See https://github.com/Expensify/App/issues/13802 */
+
     const lineHeight = useMemo(() => {
         if (Browser.isSafari() && _.isArray(props.inputStyle)) {
             const lineHeightValue = _.find(props.inputStyle, (f) => f.lineHeight !== undefined);
             if (lineHeightValue) {
                 return lineHeightValue.lineHeight;
             }
-        } else if (Browser.isSafari() || Browser.isMobileChrome()) {
```

Comment:
@ayazalavi We had an issue on Chrome as well. If we remove the condition, won't it recreate the problem on mWeb/Chrome?

Reply:
@sobitneupane issue is not reported for android chrome by the bug reporter. Please cross verify it on your end as well.

Reply:
But since we have known the issue exists in android chrome as well, our goal should be to handle the issue on all platforms.

Reply:
Can you check if you can see the issue on android chrome? It is working fine now on my end.

Reply:
The issue is still reproducible on mWeb/Chrome.

https://github.com/Expensify/App/assets/25876548/e9b00746-8b53-433e-9b39-d9727d917d7d

Reply:
@sobitneupane I have made updates, this issue must be fixed now on mobile chrome as well. Kindly make sure to test other things if something is broken then let me know

Reply:
@ayazalavi Please make sure to make thorough review in your end before requesting review. Also, after making major update or any platform specific changes, make sure to update the screen recordings.

Reply:
Sure let me do that.

Reply:
it is working fine on my end.

https://github.com/Expensify/App/assets/767439/fb17e3c1-77ae-438d-aa78-370a56a5feb8

https://github.com/Expensify/App/assets/767439/9f7b4f03-c8d1-42db-ac6d-155e94642f70

But you need to check as per platform requirement as well. I might have missed something.